### PR TITLE
Diagnosis server side code

### PR DESCRIPTION
**Pull request by mithun-grmtech**

### CODE REVIEW

File: servers/lumen/app/Http/Controllers/DiagnosisController.php [hunk 1]
```diff
@@ -0,0 +1,46 @@
...
+use Illuminate\Http\Request;
+use Ramsey\Uuid\Uuid;
+use DB;
+use Predis\Autoloader;
+
+class DiagnosisController extends Controller
+{
+    public function getAllDiagnosis()
+    {
+        return response()->json(Diagnosis::all());
```
Comment (rajgrmtech) [inline servers/lumen/app/Http/Controllers/DiagnosisController.php:15]: There should be get all the data from temporal DB.
Ref: reminderController.php

Comment (mithun-grmtech) [reply servers/lumen/app/Http/Controllers/DiagnosisController.php:15]: I don't understand the temporal DB code for the reminder controller

File: servers/lumen/app/Http/Controllers/DiagnosisController.php [hunk 2]
```diff
@@ -0,0 +1,46 @@
...
+
+use App\Diagnosis;
+use Illuminate\Http\Request;
+use Ramsey\Uuid\Uuid;
+use DB;
+use Predis\Autoloader;
+
+class DiagnosisController extends Controller
+{
+    public function getAllDiagnosis()
```
Comment (vikaskedia) [inline servers/lumen/app/Http/Controllers/DiagnosisController.php:13]: the name is not correct

**Review by vikaskedia — CHANGES_REQUESTED** [review]

**Comment by mithun-grmtech** [comment]
Create a new pull request [132](https://github.com/savantcare/emr/pull/132)

**Pull request closed without merging**